adage/analyze/migrations/0009_auto_20170503_1700.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should also be code reviewed, because even though it is a migration file, it is a data migration file that I created manually to handle the non-null participation_type FK constraint when no ParticipationTypes have been added to the database.